isis/src/base/objs/CSMCamera/CSMCamera.h

```diff
@@ -116,6 +116,10 @@ namespace Isis {
       virtual double RightAscension();
       virtual double Declination();
 
+      std::vector<int> getParameterIndices(csm::param::Set paramSet) const;
+      void applyParameterCorrection(int index, double correction);
+      double getParameterCovariance(int index1, int index2);
```

Comment:
Note that these are not exposed via the Camera API so we have to dynamic cast to get to them.